### DISCUSSION TITLE
MC526131: Retirement of custom default result sources in Microsoft Search for modern search experiences

### DISCUSSION THREAD

**Discussion by damianino1** [microsoft-search/pnp-modern-search#2903] [General]
Hello,

will my modern search webpart stop to work if I configured it with SharePoint Search due to this Microsoft's MC526131 change [https://m365admin.handsontek.net/retirement-of-custom-default-result-sources-in-microsoft-search-for-modern-search-experiences/](url)

Will I have to re-build it in other way? 

Please help or share your experiences with me. Thank you in advance!

<img width="320" alt="image" src="https://user-images.githubusercontent.com/102507233/231380002-e448d685-b2b1-4d7e-8664-aa521717af73.png">

**Comment by kasperbolarsen** [comment]
See https://www.techmikael.com/2023/03/retirement-of-custom-default-result.html?m=1

But the short answer is no.

**Reply by damianino1** [reply]
Great, thank you!
